src/hutil.ml

```diff
@@ -46,7 +46,7 @@ value print_link_to_welcome = gen_print_link_to_welcome (fun () -> ());
 value header_without_http conf title = do {
   Wserver.printf "<!DOCTYPE html>\n";
   Wserver.printf "<head>\n";
-  Wserver.printf "  <title>\n";
+  Wserver.printf "  <title>";
```

Comment:
æsthetical > because title is short and we have one line in HTML source

`<title>blabla</title>` and not
```
<title>
title</source>
```